### DISCUSSION TITLE
Another close to all in one version

### DISCUSSION THREAD

**Discussion by bolle732** [Enkidu-6/tor-ddos#56] [Ideas]
Very nice to see, that all the different files for the different setups has been unified. In the meantime, I've made my own version of a script based on your work. So I've stolen a lot of code from your work... Maybe you can take some parts back for you...

Here's my version:
https://github.com/bolle732/tor-ddos-firewall-rules

Cheers

**Comment by Enkidu-6** [comment]
Nice, thanks. I'll check it out.
